packages/imperative/__tests__/__integration__/cmd/__tests__/integration/cli/auth/__scripts__/auth_login_cmd_line_cert.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+echoVal=${1:?"First parm (echoVal) is required."}
+baseCertFile=${2:?"Second parm (baseCertFile) is required."}
+baseCertKey=${3:?"Third parm (baseCertKey) is required."}
+
+# include exitOnFailure function
+myScriptDir=`dirname $0`
+. $myScriptDir/exitOnFailure.sh
+
+# copy our config file and certificate files
+resourceDir=$myScriptDir/../__resources__
+cp $resourceDir/base_cert.config.json .
+exitOnFailure "Failed to copy config file." $?
+
+cp $resourceDir/$baseCertFile .
+exitOnFailure "Failed to copy certificate file." $?
+
+cp $resourceDir/fakeKey.key .
```

Comment:
Should this be copying `$resourceDir/$baseCertKey`?

Reply:
Seems like we would want to use the required `baseCertKey` here 😋

Reply:
Yes. I just forgot to use the supplied variable. Thanks.